docs/asset-loader.md

```diff
@@ -0,0 +1,56 @@
+# Webpack Asset Loader
+
+This feature makes it easy to detect and load assets generated by Webpack, including those served from local `webpack-dev-server` instances.
+
+## Usage
+
+The Asset Loader library is designed to work in conjunction with a Webpack configuration (such as those created with the presets in [@humanmade/webpack-helpers](https://github.com/humanmade/webpack-helpers)) which generate an asset manifest file. This manifest associates asset bundle names with either URIs pointing to asset bundles on a running DevServer instance, or else local file paths on disk.
+
+### `Asset_Loader\register_asset()` and `Asset_Loader\enqueue_asset()`
+
+`Asset_Loader` provides a set of methods for reading in this manifest file and registering a specific resource within it to load within your WordPress website. The primary public interface provided by this plugin is a pair of methods, `Asset_Loader\register_asset()` and `Asset_Loader\enqueue_asset()`. To register a manifest asset call one of these methods inside actions like `wp_enqueue_scripts` or `enqueue_block_editor_assets`, in the same manner you would have called the standard WordPress `wp_register_script` or `wp_enqueue_style` functions.
```

Comment:
"`Asset_Loader` provides a set of methods for reading _this_ manifest file" instead of "in this"?

Reply:
I think "reading in" here is literally referring to consuming the file using `file_get_contents()`, you could say you're reading it in to the program, making this change won't change the meaning and will remove any potential for distracting the reader from the content itself 👍 
```suggestion
`Asset_Loader` provides a set of methods for reading this manifest file and registering a specific resource within it to load within your WordPress website. The primary public interface provided by this plugin is a pair of methods, `Asset_Loader\register_asset()` and `Asset_Loader\enqueue_asset()`. To register a manifest asset call one of these methods inside actions like `wp_enqueue_scripts` or `enqueue_block_editor_assets`, in the same manner you would have called the standard WordPress `wp_register_script` or `wp_enqueue_style` functions.
```